Packs/Orca/Integrations/Orca/Orca.py

```diff
@@ -27,9 +30,16 @@ def validate_api_key(self) -> str:
                 url_suffix="/rules/query/alerts", data={},
                 timeout=ORCA_API_TIMEOUT
             )
-        except Exception:
-            return invalid_token_string
-        if response.get("status") != "success":
+            if response.get("status") != "success":
+                return response.get("error") or invalid_token_string
```

Comment:
This logic is needed to improve the validation error message.
The API returns the reason why the validation failed, and it would be good to show this message to the user.

Reply:
This custom error handler is needed to display the correct error message to the user
Example: <img width="577" alt="image" src="https://user-images.githubusercontent.com/90267674/231201112-d5fb247e-1cbc-48a9-8cac-c24ac78d140b.png">

Not sure how to do it with the return_error() function.

Reply:
I'm sorry, i don't understand your point

Reply:
changed to use `return_error()`

Reply:
did it work as you expected it to work after change?

Reply:
Yes, I've checked it `return_error(message="")` works is i expected.